src/routes/voteOnSponsorTime.js

```diff
@@ -194,41 +194,89 @@ async function voteOnSponsorTime(req, res) {
-                let webhookURL = null;
-                if (voteTypeEnum === voteTypes.normal) {
-                    webhookURL = config.discordReportChannelWebhookURL;
-                } else if (voteTypeEnum === voteTypes.incorrect) {
-                    webhookURL = config.discordCompletelyIncorrectReportWebhookURL;
-                }
-    
-                if (config.youtubeAPIKey !== null && webhookURL !== null) {
-                    YouTubeAPI.videos.list({
-                        part: "snippet",
-                        id: submissionInfoRow.videoID
-                    }, function (err, data) {
-                        if (err || data.items.length === 0) {
-                            err && logger.error(err);
-                            return;
-                        }
-                        
+        // Get video ID
+        let submissionInfoRow = db.prepare('get', "SELECT s.videoID, s.userID, s.startTime, s.endTime, s.category, u.userName, " +
+            "(select count(1) from sponsorTimes where userID = s.userID) count, " +
+            "(select count(1) from sponsorTimes where userID = s.userID and votes <= -2) disregarded " +
+            "FROM sponsorTimes s left join userNames u on s.userID = u.userID where s.UUID=?",
+        [UUID]);
+
+        let userSubmissionCountRow = db.prepare('get', "SELECT count(*) as submissionCount FROM sponsorTimes WHERE userID = ?", [nonAnonUserID]);
+
+        if (submissionInfoRow !== undefined && userSubmissionCountRow != undefined) {
+            let webhookURL = null;
+            if (voteTypeEnum === voteTypes.normal) {
+                webhookURL = config.discordReportChannelWebhookURL;
+            } else if (voteTypeEnum === voteTypes.incorrect) {
+                webhookURL = config.discordCompletelyIncorrectReportWebhookURL;
+            }
+
+            if (config.youtubeAPIKey !== null && (webhookURL !== null || config.webhooks.size !== 0)) {
+                YouTubeAPI.videos.list({
+                    part: "snippet",
+                    id: submissionInfoRow.videoID
+                }, function (err, data) {
+                    if (err || data.items.length === 0) {
+                        err && console.log(err);
+                        return;
+                    }
+                    let isUpvote = incrementAmount > 0
+                    // Send custom webhooks
+                    if (config.webhooks.size !== 0) {
+                        logger.debug("Dispatching webhooks");
+                        config.webhooks.forEach(customWebhook => {
+                            let customWebhookURL = customWebhook.url;
+                            let scopes = customWebhook.scopes;
+                            let key = customWebhook.key;
+                            if ((!isUpvote && !scopes.includes("vote.down")) || (isUpvote && !scopes.includes("vote.up"))) {
+                                return;
+                            }
```

Comment:
Instead of returning, I think it would make more sense if you flipped the if statement.